Joon/src/converter.py

```diff
@@ -128,67 +145,44 @@ def convert(info):
     """
 
     # parameters guaranteed to be present in Maple code
-    exp_type, category, parameters, lhs, label, booklabel, constraints = "", "", "", "", "", "", ""
+    exp_type, category, lhs, booklabel, constraints = "", "", "", "", ""
     # conditional parameters
     general, factor, begin, even, odd, front = "", "", "", "", "", ""
 
     for param in info:  # initializes all variables used
-        if param == 'exp_type':
-            exp_type = info['exp_type']
-
-        elif param == 'category':
-            category = info['category']
-
-        elif param == 'parameters':
-            parameters = info['parameters']
-
-        elif param == 'lhs':
+        if param == 'lhs':
             lhs = translate(info['lhs'])
 
-        elif param == 'label':
-            label = info['label']
-
-        elif param == 'booklabel':
-            booklabel = info['booklabel']
-
         elif param == 'constraints':
             constraints = translate(info['constraints'])
 
-        elif param == 'general':
-            general = info['general']
-
         elif param == 'factor':
             factor = translate(info['factor'])
 
-        elif param == 'begin':
-            begin = info['begin']
-
-        elif param == 'even':
-            even = info['even']
-
-        elif param == 'odd':
-            odd = info['odd']
-
         elif param == 'front':
             front = translate(info['front'])
 
-    result = "\\begin{equation*}\\tag{"+booklabel+"}\n  "+lhs+"\n  = "
+        else:
+            exec param + ' = info["' + param + '"]'
```

Comment:
is this exec required
https://www.quantifiedcode.com/app/issue_class/5zamf0Zq
My feeling is that this is not optimal.